### ISSUE TITLE
Not updating skins?

### ISSUE THREAD

**Issue by iWanheda** [advancedfx/nSkinz#14]
For me it's not updating skins or knives, anything at all, update broke it?

----

Edit by @dtugend:

## Users coming from the original nSkinz:

This fork is originally meant for demos / movie makers and can replace multiple players by XUID, so you will need to enter your XUID in the XUID field!

### Getting your XUID:
You can get your own XUID after starting the game from the game's console, there should be a message like this
```
SDR RelayNetworkStatus:  avail=Attempting  config=OK  anyrelay=Attempting   (Performing ping measurement)
AuthStatus (steamid:12345678901234567):  OK  (OK)
```
12345678901234567 would be the XUID.

**Comment by anTTTTT1** [comment]
same issue dude

**Comment by dtugend** [comment]
An updated version has beel already released 8 days ago (nSkinz-advancedfx-v0.1.11):
https://github.com/advancedfx/nSkinz/releases

There's no update notifcation, because that one still points at original author's GitHub.

**Comment by iWanheda** [comment]
Oh so the round start is it an issue or? ...

**Comment by iWanheda** [comment]
Actually, even after a round restart the skin or knife never gets applied. And yes I'm using latest (which you sent), video of the issue:
https://b.catgirlsare.sexy/gV6ArmOV_n_i.mp4

**Comment by dtugend** [comment]
1) Did you update the nSkinz.DLL to the latest version from the release page as said?

2) did you specify the player's XUID and is it the correct one? (compare with `mirv_listentities isPlayer=1`

**Comment by iWanheda** [comment]
> 1. Did you update the nSkinz.DLL to the latest version from the release page as said?
> 2. did you specify the player's XUID and is it the correct one? (compare with `mirv_listentities isPlayer=1`

Yes I updated to the latest version, and I didn't know you needed to update XUID, will try now! (Will edit the post with the result)

**Comment by dtugend** [comment]
advancedfx-nSkinz is for demos / moviemaking, so you need to set the XUID of the players you want to change, unlike the original nSkinz, which is only for local player.

**Comment by iWanheda** [comment]
> advancedfx-nSkinz is for demos / moviemaking, so you need to set the XUID of the players you want to change, unlike the original nSkinz, which is only for local player.

Oh shit, I thought it was for localplayer, is it easy to make it local only? I have some knowledge but long time no C++, also is `mirv_listentities isPlayer=1` a command or what? Can't seem to execute it, what is it?

**Comment by dtugend** [comment]
This is meant to be used with https://github.com/advancedfx/advancedfx, which works in demos only.

Maybe look at the original non-demo https://github.com/namazso/nSkinz by the original author, this fork is meant for demos / game movie makers.

**Comment by dtugend** [comment]
Ah I see the problem now, the original author put this into their repo :/
![image](https://user-images.githubusercontent.com/3610128/135725038-3ce767f6-2fdd-40e0-900a-43227e59d7aa.png)

**Comment by dtugend** [comment]
As a workaround you can put your own XUID, I am not sure how to get it without getting it with HLAE from a demo though :S
I will see if I can add s.th. to easily get the XUID (a button) or to use localplayer if XUID is 0 when I have time.

**Comment by dtugend** [comment]
You can get your own XUID after starting the game from the game's console, there should be a message like this
```
SDR RelayNetworkStatus:  avail=Attempting  config=OK  anyrelay=Attempting   (Performing ping measurement)
AuthStatus (steamid:12345678901234567):  OK  (OK)
```
12345678901234567 would be the XUID.

**Comment by dtugend** [comment]
I edited your first post above with the workaround for now, since more people will probably run into the same issue coming from original nSkinz version, I hope it's ok.

**Comment by iWanheda** [comment]
Thank you so much sir! Lots of love ❤️

**Comment by iWanheda** [comment]
@dtugend As a fix for not updating skins instantly and have to wait a round or restart the map you can follow this post: https://www.unknowncheats.me/forum/2172816-post1.html

It fixed it for me, thanks!

**Comment by dtugend** [comment]
Thanks. I am not 100% sure but it actually updates for me instantly since the last one or two versions (?) (at least when you set the correct XUID). It's just I copied and pasted the old note about it from the old release accidentally on the release page.

**Comment by iWanheda** [comment]
@dtugend 

Sorry to post here again, but can I get your discord?

**Comment by dtugend** [comment]
https://discord.gg/NGp8qhN
